preprocessing/tabular_data/feature_selection_tabular_data.ipynb

```diff
@@ -0,0 +1,351 @@
+{
```

Comment:
<p>In the following cell, we rank each feature based on corresponding importance score.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1664/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>